src/hooks/useAllServices.ts

```diff
@@ -36,7 +41,7 @@ const handleBundleResponse = (bundle: { data: Omit<BundleNavigation, 'id' | 'tit
     // regular NavItem
     return [...acc, rest];
   }, []);
-  return flatLinks.flat();
+  return { id: bundle.data.id as string, title: bundle.data.title as string, links: (flatLinks || []).flat() };
```

Comment:
If the id and title cannot be `undefined` we should make sure to properly define the input type so we don't need to typecast as much.

Reply:
The typecast is not even required here.